src/main/java/org/team1540/robot2023/Robot.java

```diff
@@ -69,6 +84,10 @@ public void robotPeriodic() {
      */
     @Override
     public void disabledInit() {
+        new WaitCommand(5)
+                .andThen(() -> robotContainer.drivetrain.setNeutralMode(NeutralMode.Coast))
```

Comment:
Is it necessary to wait 5 seconds before setting to coast when disabling?

Reply:
This is something we ran into last year, immediately stopping if the robot was driving quickly isn't ideal, letting it coast at the end of matches tends to be better